dbt_copilot_helper/commands/dns.py

```diff
@@ -741,5 +746,212 @@ def assign(app, domain_profile, project_profile, svc, env):
+                                        "Are you sure you wish to delete the domain ", fg="yellow"
+                                    )
+                                    + click.style(f"{cdn_domain}?", fg="white", bold=True),
+                                ):
+                                    # Exit if specified domain doesn't exist.
+                                    if not cdn_domain in cond["Values"]:
+                                        click.echo(
+                                            click.style(
+                                                f"{cdn_domain} doesn't exists, exiting", fg="red"
+                                            ),
+                                        )
+                                        save = False
+                                        break
+
+                                    # At present at least 1 domain needs to be on the listener.
+                                    if len(cond["Values"]) == 1 and force != True:
+                                        click.echo(
+                                            click.style(
+                                                f"{cdn_domain} is the only domain configured on the "
+                                                + "LoadBalancer, exiting",
+                                                fg="red",
+                                            ),
+                                        )
+                                        save = False
+                                        break
+
+                                    click.echo(
+                                        click.style(f"deleting {cdn_domain}", fg="green"),
+                                    )
+                                    cond["HostHeaderConfig"]["Values"].remove(cdn_domain)
+                                # Exit if not sure.
+                                else:
+                                    save = False
+                                    break
+                            else:
+                                # When adding exit if domain already exists.
+                                if cdn_domain in cond["Values"]:
+                                    click.echo(
+                                        click.style(
+                                            f"{cdn_domain} already exists, exiting", fg="red"
+                                        ),
+                                    )
+                                    save = False
+                                    break
+
+                                cond["HostHeaderConfig"]["Values"].append(cdn_domain)
+
+                            # Remove unwanted config values as not needed in update.
+                            cond.pop("Values", None)
+
+                        if cond["Field"] == "path-pattern":
+                            cond.pop("PathPatternConfig", None)
+
+                    if save is True:
```

Comment:
just `if save:` should suffice here shouldn't it?

Reply:
agreed, school boy mistake.